### DISCUSSION TITLE
Enabling output for use with Orbuculum (Cortex SWO tracing and debugging)

### DISCUSSION THREAD

**Discussion by chmorgan** [zephyrproject-rtos/zephyr#67295] [General]
https://github.com/orbcode/orbuculum

Trying to figure out how, or if someone else, has configured zephyr to output data for orbuculum to process. Nothing comes in searches although it seems like https://docs.zephyrproject.org/latest/services/tracing/index.html and https://docs.zephyrproject.org/latest/samples/subsys/debug/debugmon/README.html#debugmon-sample are similar.

Am I looking to specific and missing more generic SWO output support? I See CONFIG_COREXT_M_DWT but not seeing many instances of this config option being used.

**Comment by sadguitarius** [comment]
@chmorgan did you ever find a solution to this? I’m currently stuck trying to do the same thing
